pkg/sdk/poc/generator/templates/sub_templates/implementation_functions.tmpl

```diff
@@ -16,10 +16,8 @@
     {{ else if eq .Name "ShowByID" }}
         func (v *{{ $impl }}) ShowByID(ctx context.Context, id {{ .ObjectInterface.IdentifierKind }}) (*{{ .ObjectInterface.NameSingular }}, error) {
             request := NewShow{{ .ObjectInterface.NameSingular }}Request()
-            {{- range .ShowByIDFiltering }}
-                {{- if not (eq .Name "NoFiltering") -}}.
-                    {{ .WithFiltering }}
-                {{- end }}
+            {{- range .ShowByIDFiltering }}.
```

Comment:
question: is the dot after `}}` needed?
ahh it's the dot after every filtering operation? e.g. `WithLike(...). <- this one?`

Reply:
yup exactly that one, it is needed to call more than one filtering option e.g. Like and In